ext/bcmath/php_bcmath.h

```diff
@@ -30,7 +30,7 @@ ZEND_BEGIN_MODULE_GLOBALS(bcmath)
 	bc_num _zero_;
 	bc_num _one_;
 	bc_num _two_;
-	int bc_precision;
+	size_t bc_precision;
```

Comment:
Revert this for the time being, as I can imagine there is an issue during the INI set function which casts this to a an ``int`` causing issues.

Reply:
Thank you for fixing the failing test.